ruby_on_rails/app_initialisation.md

```diff
@@ -13,7 +13,11 @@
 * [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
 You can do this with `gem update rails`. Beware of beta versions.
 
-* Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
+* Start a new Rails project using
+```
+rails new [project-name] --database=postgresql --skip_ci --no-skip-test --skip_action_mailbox -m https://raw.githubusercontent.com/renuo/applications-setup-guide/main/ruby_on_rails/template.rb
```

Comment:
Typo?
```suggestion
rails new [project-name] --database=postgresql --skip-ci --no-skip-test --skip_action_mailbox -m https://raw.githubusercontent.com/renuo/applications-setup-guide/main/ruby_on_rails/template.rb
```

Reply:
By the looks of the docs it seems so. Also for `--skip_action_mailbox`, it should be `--skip-action-mailbox`:
https://github.com/rails/rails/issues/44325
https://guides.rubyonrails.org/command_line.html#rails-new

I don't find a `--skip-ci` flag in the docs though - where does this come from?

Reply:
Right! That's why also mailbox was not skipped! 😆